crates/sui-framework/packages/deepbook/sources/math.move

```diff
@@ -62,8 +61,81 @@ module deepbook::math {
         (is_round_down, result)
     }
 
+    public fun count_leading_zeros(x: u128): u8 {
```

Comment:
```suggestion
    public(friend) fun count_leading_zeros(x: u128): u8 {
```

This will give us the flexibility to change or delete this in the future.

Reply:
Ok！